pyServer/manifests/windows/min-diagnostic

```diff
@@ -3,3 +3,9 @@ copy,/WindowsAzure/Logs/TransparentInstaller.log
 echo,### Registry Hives ###
 copy,/Windows/System32/config/SOFTWARE
 copy,/Windows/System32/config/SYSTEM
```

Comment:
Does it make sense to do something like `copy,/Windows/System32/config/SYSTEM*` or `copy,/Windows/System32/config/SYSTEM.LOG*`  instead?  Or do you specifically want just LOG1 & LOG2?

Reply:
From what I understand, Windows only ever writes to LOG1 and LOG2 files specifically.